### DISCUSSION TITLE
Is cross-compiling ARM on Linux not supported?

### DISCUSSION THREAD

**Discussion by JAEWOOKe** [microsoft/onnxruntime#6862] [Other Q&A]
Dear all,

Hi, I'm interested in cross-compiling ARM on Linux but it seems to be not supported.

Is it possible to cross-compile through Linux (Ubuntu16.04) for ARM?

Thanks
Jake

**Comment by faxu** [comment]
See: https://www.onnxruntime.ai/docs/how-to/build.html#arm

**Comment by snnn** [comment]
It works. I have a daily build pipeline to test it.

**Reply by JAEWOOKe** [reply]
Thanks for replying. @faxu, it has been a huge help.

I have one more question. 

I've successfully mounted PYNQ img at /mnt/pi and trying to follow the steps.

`cmake -Donnxruntime_GCC_STATIC_CPP_RUNTIME=ON -DCMAKE_BUILD_TYPE=Release -Dprotobuf_WITH_ZLIB=OFF -DCMAKE_TOOLCHAIN_FILE=/home/jwlee/onnx-runtime/tool.cmake -Donnxruntime_ENABLE_PYTHON=ON -DPYTHON_EXECUTABLE=/mnt/pi/usr/bin/python3 -Donnxruntime_BUILD_SHARED_LIB=OFF -Donnxruntime_DEV_MODE=OFF -DONNX_CUSTOM_PROTOC_EXECUTABLE=/home/jwlee/protoc3/bin/protoc "-DPYTHON_INCLUDE_DIR=/mnt/pi/usr/include;/mnt/pi/usr/include/python3.6m" -DNUMPY_INCLUDE_DIR=/mnt/pi/usr/include/python3.6m/numpy -DPYTHON_LIBRARY=/mnt/pi/usr/lib/python3.6/config-3.6m-aarch64-linux-gnu ../cmake
`
However, I'm facing the following problem.

` /lib/ld-linux-aarch64.so.1: No such file or directory`

Can you give me any advice?

Thanks,
Jaewook

**Reply by snnn** [reply]
I need more info to tell you.  Could you please give me the logs around "/lib/ld-linux-aarch64.so.1: No such file or directory". I need to know who tried to invoke an ARM executable file on your build host(which is x86).
